docs/source/examples/sentiment.rst

```diff
@@ -0,0 +1,359 @@
+Sentiment Analysis with Recursive Neural Network
+*************************************************
+
+.. currentmodule:: chainer
+
+0. Introduction
+================
+
+In this tutorial, we will use the Recursive Neural Network to analyze sentiment in sentences.
+
+Sentiment analysis is one of the major tasks of Natural Language Processing (NLP),
+which identifies writers’ sentiments in sentences. When expressing sentiment,
+we basically uses labels whether it is positive or negative. For example,
+in the case of the dataset used this time, emotions are expressed in 5 classes
+like 1(really negative), 2(negative), 3(neutral), 4(positive), 5(really positive).
+
+.. figure:: ../../image/sentiment/sentiment-treebank.png
+    :scale: 100%
+
+    cited from [1]
+
+Sentiment analysis is implemented with Recursive Neural Network. Recursive Neural
+Network is a recursive neural net with a tree structure. NLP often expresses
+sentences in a tree structure, Recursive Neural Network is often used in NLP.
+First, we explain the training method of Recursive Neural Network without
+mini-batch processing. After that, as a advanced story, we explain the training
+method of mini-batched Recursive Neural Network.
```

Comment:
of a mini-batched

Reply:
should be "method of mini-batched Recursive Neural Networks" ?